### PR TITLE
[13.0] [MIG] account_payment_return_import_iso20022: Migration to v13

### PR DESCRIPTION
**Pull request by victoralmau**
@Tecnativa TT19693

Please @pedrobaeza review it

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@sbidoul why this weird ping to a non maintainer?

File: account_payment_return_import_iso20022/__manifest__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "summary": """
         This addon allows to import payment returns from ISO 20022 files
         like PAIN or CAMT.""",
-    "version": "12.0.1.1.0",
+    "version": "13.0.1.1.0",
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/__manifest__.py]:
```suggestion
    "version": "13.0.1.0.0",
```

File: account_payment_return_import_iso20022/__manifest__.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
...
     "development_status": "Production/Stable",
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/__manifest__.py]: You can change it to mature:
```suggestion
    "development_status": "Mature",
```

File: account_payment_return_import_iso20022/tests/test_import_iso20022.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 
 class TestImport(TestPaymentReturnFile):
-    """Run test to import payment return import."""
+    # Run test to import payment return import
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/tests/test_import_iso20022.py]: Don't change docstring by comments

File: account_payment_return_import_iso20022/tests/test_import_iso20022.py [hunk 4]
```diff
@@ -28,14 +28,13 @@ def setUpClass(cls):
...
                 "type": "bank",
-                "update_posted": True,
                 "bank_account_id": cls.acc_bank.id,
             }
         )
         cls.journal.bank_account_id = cls.acc_bank
 
     def test_payment_return_import_pain(self):
-        """Test correct creation of single payment return."""
+        # Test correct creation of single payment return
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/tests/test_import_iso20022.py]: Restore it

File: account_payment_return_import_iso20022/tests/test_import_iso20022.py [hunk 5]
```diff
@@ -71,7 +70,7 @@ def test_payment_return_import_pain_invalid(self):
             )
 
     def test_zip_import_pain(self):
-        """Test import of multiple statements from zip file."""
+        # Test import of multiple statements from zip file
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/tests/test_import_iso20022.py]: Restore it

File: account_payment_return_import_iso20022/tests/test_import_iso20022.py [hunk 6]
```diff
@@ -81,7 +80,7 @@ def test_zip_import_pain(self):
         )
 
     def test_payment_return_import_camt(self):
-        """Test correct creation of single payment return."""
+        # Test correct creation of single payment return
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/tests/test_import_iso20022.py]: Restore it

File: account_payment_return_import_iso20022/tests/test_import_iso20022.py [hunk 7]
```diff
@@ -117,7 +116,7 @@ def test_payment_return_import_camt_invalid(self):
             )
 
     def test_zip_import_camt(self):
-        """Test import of multiple statements from zip file."""
+        # Test import of multiple statements from zip file
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/tests/test_import_iso20022.py]: Restore it

File: account_payment_return_import_iso20022/wizard/camt_parser.py [hunk 8]
```diff
@@ -12,11 +12,11 @@
 
 
 class CamtParser(object):
-    """Parser for CAMT Bank to Customer Debit Credit Notification."""
+    # Parser for CAMT Bank to Customer Debit Credit Notification
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/camt_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/camt_parser.py [hunk 9]
```diff
@@ -12,11 +12,11 @@
...
 
     @staticmethod
     def parse_amount(ns, node):
-        """Parse element that contains Amount."""
+        # Parse element that contains Amount
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/camt_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/camt_parser.py [hunk 10]
```diff
@@ -29,7 +29,7 @@ def parse_amount(ns, node):
 
     @staticmethod
     def parse_date(ns, node):
-        """Parse element that contains date."""
+        # Parse element that contains date
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/camt_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/camt_parser.py [hunk 11]
```diff
@@ -53,9 +53,7 @@ def add_value_from_node(ns, node, xpath_str, obj, key, join_str=None):
                 break
 
     def parse_transaction_details(self, ns, node, transaction):
-        """
-        Parse transaction details.
-        """
+        # Parse transaction details
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/camt_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/camt_parser.py [hunk 12]
```diff
@@ -87,9 +85,7 @@ def parse_transaction_details(self, ns, node, transaction):
         )
 
     def parse_transactions(self, ns, node, transactions):
-        """
-        Parse transactions (entry) node.
-        """
+        # Parse transactions (entry) node
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/camt_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/camt_parser.py [hunk 13]
```diff
@@ -103,9 +99,7 @@ def parse_transactions(self, ns, node, transactions):
         return transactions
 
     def parse_payment_returns(self, ns, node):
-        """
-        Parse entry node.
-        """
+        # Parse entry node.
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/camt_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/pain_parser.py [hunk 14]
```diff
@@ -8,10 +8,10 @@
 
 
 class PainParser(object):
-    """Parser for SEPA Direct Debit Unpaid Report import files."""
+    # Parser for SEPA Direct Debit Unpaid Report import files
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/pain_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/pain_parser.py [hunk 15]
```diff
@@ -8,10 +8,10 @@
...
 
     def parse_amount(self, ns, node):
-        """Parse element that contains Amount and CreditDebitIndicator."""
+        # Parse element that contains Amount and CreditDebitIndicator
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/pain_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/pain_parser.py [hunk 16]
```diff
@@ -21,7 +21,7 @@ def parse_amount(self, ns, node):
         return amount
 
     def parse_date(self, ns, node):
-        """Parse element that contains date."""
+        # Parse element that contains date
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/pain_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/pain_parser.py [hunk 17]
```diff
@@ -44,7 +44,7 @@ def add_value_from_node(self, ns, node, xpath_str, obj, key, join_str=None):
                 break
 
     def parse_transaction_details(self, ns, node, transaction):
-        """Parse transaction details."""
+        # Parse transaction details
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/pain_parser.py]: Restore it

File: account_payment_return_import_iso20022/wizard/pain_parser.py [hunk 18]
```diff
@@ -58,7 +58,7 @@ def parse_transaction_details(self, ns, node, transaction):
         )
 
     def parse_transaction(self, ns, node, transaction):
-        """Parse transaction (entry) node."""
+        # Parse transaction (entry) node
```
Comment (pedrobaeza) [inline account_payment_return_import_iso20022/wizard/pain_parser.py]: Restore it

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
I haven't continued, but restore all the docstrings. See https://www.python.org/dev/peps/pep-0257/ for knowing more about docstring that seems that you don't like, hehe.

**Review by pedrobaeza — APPROVED** [review]
You may want to rebuild README file with the OCA tool

**Comment by victoralmau** [comment]
Sorry @pedrobaeza  but i'not regenerate readme yet with OCA tool. Can you regenerate it when oca bot apply the changes?

**Comment by pedrobaeza** [comment]
OK, bot can do it on merge. It was just for seeing the changes directly.

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**